projects/plugins/jetpack/extensions/blocks/premium-content/_inc/subscription-service/class-token-subscription-service.php

```diff
@@ -177,6 +177,10 @@ private function token_from_cookie() {
 	 * @return void
 	 */
 	private function set_token_cookie( $token ) {
+		if ( defined( 'TESTING_IN_JETPACK' ) && TESTING_IN_JETPACK ) {
```

Comment:
Otherwise I get a 
```
Warning: Cannot modify header information - headers already sent (output started at script:line[...])
```
Headers are sent in bootstrap.php

Reply:
I see